client/my-sites/site-settings/site-tools/index.jsx

```diff
@@ -206,9 +205,7 @@ export default compose( [
 				sourceSlug: siteSlug,
 			} );
 
-			const showStartSiteTransfer =
-				isEnabled( 'yolo/site-transfers-i1' ) &&
```

Comment:
Does it appear anywhere else in the codebase?

Reply:
Yes it does! I [asked on Slack what is the best approach](p1688391075613119/1688384958.548809-slack-C04GESRBWKW) and decided to first make it visible and then clean up the feature flag completely in a few days in case we need to hide it urgently again for some reason.

Reply:
@katinthehatsite To clarify, I meant to leave this bit of code in place, and add `"yolo/site-transfers-i1": true` to all config files.